### ISSUE TITLE
juce_PixelFormats.h: cannot bind packed field

### ISSUE THREAD

**Issue by dvzrv** [jpcima/ADLplug#54]
Hi! I would like to bring adlplug to the [community] repository on Arch Linux.
However, when building 1.0.1, I'm running into build errors with the juce components:
[adlplug-1.0.1-build.txt](https://github.com/jpcima/ADLplug/files/3331155/adlplug-1.0.1-build.txt)

I'm currently building with gcc 9.1.0.

I wonder, whether I'm missing something. Any help debugging this would be greatly appreciated!

**Comment by jpcima** [comment]
Hi, this has likely been fixed in JUCE upstream.
See issue: https://github.com/WeAreROLI/JUCE/issues/522 and [patch](https://github.com/WeAreROLI/JUCE/commit/4e0adb2af8b424c43d22bd431011c9a6c57d36b6.diff)

I'll give it a try myself sometime later and update the AUR package with this.

**Comment by dvzrv** [comment]
Ah, seems gcc 9.1.0 is the problem for adlplug here!
We have 8.3.0 still in the repositories and using it, adplug builds successfully!

I'm changing the name of this issue to reflect this.

**Comment by jpcima** [comment]
Exact, I have noticed a little ago when I restarted a build for reasons I since forgot.
My working setup is based on clang, except for automatic and cross-compiled builds, based on older gcc.

**Comment by dvzrv** [comment]
The patch seems to work for 1.0.1 so far. Thanks for the hint!

**Comment by dvzrv** [comment]
The patch is still required for 1.0.2 btw. It would be great if it could be included! :)

**Comment by jpcima** [comment]
Agreed, but I don't touch juce at the moment, and don't want either to add into the already existing fork.
The proper job to do is upgrading all to juce 6, but it's not on radar for anytime very soon.
